STDOUT

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Negative. Do those get autogenerated as a normal process of the app running? 

(Maybe worth adding them to the gitignore?)